release-notes-3.4.0.md

```diff
@@ -1,9 +1,13 @@
-!not-ready-for-release!
-
 #### Version Number
 ${version-number}
 
 #### New Features
-[SCMOD-7343](https://portal.digitalsafe.net/browse/SCMOD-7060) - Audit Service has been updated to support Elasticsearch 7.2.0
+- **SCMOD-7343**: Audit Service has been updated to support Elasticsearch 7.2.0
+- **SCMOD-4930**: Removed elastic search cluster name specification
```

Comment:
SCMOD-4930 was a jira to upgrade to ES6 so I don't think this change is relevant now we are on ES7. Also from reading the change it looks like the change was made so we can no longer define the cluster name of ES for the audit service to connect to.

Reply:
@dermot-hardy ,

Please confirm whether the ES cluster name is mandatory to connect with audit service?

Reply:
@thilagasanthosh This is not what I am asking you to check. From the change it looks like we have taken away the ability to configure the cluster name we want to connect to. If this is the case then users need to be made aware of this. If it was mandatory then surely it would not work anymore with the change made?

Reply:
@dermot-hardy ,

Please clarify on Michael's question in ES cluster name.
Is this change to be notified to the user for not configuring cluster name?